Exiled.API/Features/Player.cs

```diff
@@ -1632,30 +1727,58 @@ public void AddItem(Item item, int amount)
             }
         }
 
+        /// <summary>
+        /// Add the amount of items to the player's inventory.
+        /// </summary>
+        /// <param name="item">The item to be added.</param>
+        /// <param name="amount">The amount of items to be added.</param>
+        /// <param name="identifiers">The attachments to be added to the item.</param>
+        public void AddItem(Item item, int amount, IEnumerable<AttachmentIdentifier> identifiers)
+        {
+            if (amount > 0)
+            {
+                for (int i = 0; i < amount; i++)
+                    AddItem(item, identifiers);
+            }
+        }
+
         /// <summary>
         /// Add the list of items to the player's inventory.
         /// </summary>
         /// <param name="items">The list of items to be added.</param>
-        public void AddItem(List<Item> items)
+        public void AddItem(IEnumerable<Item> items)
```

Comment:
This is a breaking change.

Reply:
The entire PR is a breaking change, therefore I think it should be accepted if we want to enhance it regarding the implementation and compatibility with regard to the Item and Attachment system.